csrc/ir_internal_nodes.h

```diff
@@ -1082,14 +1084,48 @@ class TORCH_CUDA_CU_API MmaOp : public Expr {
   }
 
   const auto& options() const {
-    return attribute(1)->as<Attribute<OptionsInMma>>()->value;
+    return attribute(ATTR_POS_OPTS)->as<Attribute<OptionsInMma>>()->value;
   }
 
   auto accStride() const {
     return options().accumulator_stride;
   }
 
   void configureOptions(MmaOptions options);
+
+  auto inputLayout() const {
```

Comment:
`OptionsInMma` already contain an `operand_layout`, we should reuse it. How about merging `MmaOpDetails` with `OptionsInMma`?

Reply:
In fact, I planned to remove `operand_layout` from `OptionsInMma` and instead rely on data stored in attributes to input layouts. But this would be in a separate PR, as I want to be sure that there are no issues with accessing inputs' layout data in runtime checks / heuristics and how MmaBuilder instance is constructed in `scheduleMatmul` and what changes does it do with an instance of `MmaOp`.

So for now I planned to keep this layout redundancy (attribute and field in `OptionsInMma`) and deal with it in next step.

Reply:
Sure, this makes sense